src/main.c

```diff
@@ -269,101 +321,109 @@ GLuint gen_cube_buffer(float x, float y, float z, float n, int w) {
         {0.5, 0.5, 0.5, 0.5},
         {0.5, 0.5, 0.5, 0.5},
         {0.5, 0.5, 0.5, 0.5},
-        {0.5, 0.5, 0.5, 0.5}
-    };
+        {0.5, 0.5, 0.5, 0.5}};
     make_cube(data, ao, light, 1, 1, 1, 1, 1, 1, x, y, z, n, w);
     return gen_faces(10, 6, data);
 }
 
-GLuint gen_plant_buffer(float x, float y, float z, float n, int w) {
+GLuint gen_plant_buffer(float x, float y, float z, float n, int w)
+{
     GLfloat *data = malloc_faces(10, 4);
     float ao = 0;
     float light = 1;
     make_plant(data, ao, light, x, y, z, n, w, 45);
     return gen_faces(10, 4, data);
 }
 
-GLuint gen_player_buffer(float x, float y, float z, float rx, float ry) {
+GLuint gen_player_buffer(float x, float y, float z, float rx, float ry)
+{
     GLfloat *data = malloc_faces(10, 6);
     make_player(data, x, y, z, rx, ry);
     return gen_faces(10, 6, data);
 }
 
-GLuint gen_text_buffer(float x, float y, float n, char *text) {
+GLuint gen_text_buffer(float x, float y, float n, char *text)
+{
     int length = strlen(text);
     GLfloat *data = malloc_faces(4, length);
-    for (int i = 0; i < length; i++) {
+    for (int i = 0; i < length; i++)
+    {
         make_character(data + i * 24, x, y, n / 2, n, text[i]);
         x += n;
     }
     return gen_faces(4, length, data);
 }
 
-void draw_triangles_3d_ao(Attrib *attrib, GLuint buffer, int count) {
+void draw_triangles_3d_ao(Attrib *attrib, GLuint buffer, int count)
+{
```

Comment:
Can you reformat the code and bring back open curly braces to its original position as it just simply increased number of changes in the file which is unnecessary.